desktop/windows/wsl.md

```diff
@@ -49,19 +44,29 @@ Docker Desktop does not require any particular Linux distros to be installed. Th
 
     To check the WSL mode, run:
 
-     `wsl.exe -l -v`
+     ```consolse
+     $ wsl.exe -l -v
+     ```
 
     To upgrade your existing Linux distro to v2, run:
 
-    `wsl.exe --set-version (distro name) 2`
+    ```console
+    $ wsl.exe --set-version (distro name) 2
+    ```
 
     To set v2 as the default version for future installations, run:
 
-    `wsl.exe --set-default-version 2`
+    ```console
+    $ wsl.exe --set-default-version 2
+    ```
 
 2. When Docker Desktop starts, go to **Settings** > **Resources** > **WSL Integration**.
 
-    The Docker-WSL integration will be enabled on your default WSL distribution. To change your default WSL distro, run `wsl --set-default <distro name>`.
+    The Docker-WSL integration is enabled on your default WSL distribution. To change your default WSL distro, run:
+    
+    ```console
+    $ wsl --set-default <distro name>
```

Comment:
This contains `<distro name>` as placeholder, so copying and running won't work; as we have an actual example already below (using `ubuntu`), perhaps we should use that as example in the code-block, and leave this one in-line